### PR TITLE
Update-tikv-dashboard

### PR DESCRIPTION
**Pull request by qiffang**
**Function**
For 2.1.0 TIKV, we use push gate way to expose metrics, but there are some metrics belong to push gate way.
For example
image
So we need to exclude these unuseful metrics in dashboard

**Test**
The expression in dashboard like this:
avg(process_resident_memory_bytes{job=~"$job"}) by (job)

I will upload the original tikv dashboard json to tidb-docker-compose repository

### CODE REVIEW

File: charts/tidb-cluster/templates/monitor-dashboard-configmap.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ binaryData:
...
...OvoGaLzKAHX9Qp3bbSI3+J0SqEvOHyXdzKZY0iSkoJTjpJSFYdlharlxkUlSrtHACCtSbu8B0AhoJPUBjQyZRq57To/LZgK+X62Dx5KTD4q4gacGDNQ7A9UMuN7m3tiBlGwSihsIEJ5IZ8sahanhgNH19ZdaDEeUjbD4Ww066nYjTsDR0cDR7NnFRb94pA7ggFNGzikHzygp4ra5N5OZFEfSo/YxrbRlFDjIXaaO2kNfkVzrqgV/c477AZNumkogBzcXSK41kkNQkZCC5EZOci16nFp3OB2rS+nHXyIagWNJGR6BY6kvHBHfJIHdksjtdgK72WuJeO5mIp6eXh8pfF0Cvo4MvrKMtZ7rHvljQ3bL61PlwdkLlDidaXui0oUSpPT6hSY0iTQ+fWOR2kQxbjJKzh8pK1xlHwOsAFYAK5RmFIxJoZBbEOghZA2iR5JR8CuRD/FrwEjPMKLuAA4QGTKInM2OdcYIIHL0ILIkhhfcESPQRYCELP4EyFCFDLEK4a9AWEkDkOCuV+qW4jgR4fz6WLe6BiIcPSLQWrXoy0X0NSGeGnhwYIY/ZdAhUq3GVAt8AD4cq49BfOszDOT1i21BIF+IB5z45pKsjF/oKxs+xOwi/Dl4DC81N7wHfjHa+94nfcckIKu1TZ/cuY9fHzq0+UGqkSYVS2HC/oU184KKU6iQZqp6BBSlZE11EbRDoMfNyXecf7JvW9xu0nkWWdVvaOdUPFq8y7mD3TXPu57rRCd/bIjHloOH99DDdzqTuPbOds0H3TSovnWfVod+9xgQ/0Sb36XHGNbls44623zpr/fkQ26MiDrI9HG0jrie/f+MnmeSlRYenP0mP1j0mmHJUoKNT96GF+IjtBDEladurZvuamU48x2qXnLG7vXvU0pgzghxsZA6ROzpEdR8uGLp1+np9e//Oum7CbxyKH06JpG1gRxvxo2AcmEjHUD6pafX3KXun6R0+ETdhsD/z0bCz+EQYXEFisFhEQLmxHHfn84iDKNkKH6bZE5bWxTAvORkoR89Yu80C00uE9SZzKapP87Tf8wST8TkMvV9lv7jfJqWJLA4OUt9n83DQfT3qAwsRjzVLLbeJX3hp+kLp+9ydpH+I+mDJlfz9PNGz5JR30eXW1CTO89974s3J6GDt69ePD+9CV+q07fWj7+E8ncxA5x98fmL/w9UwJI9UncGAA==
```
Comment (weekface) [inline charts/tidb-cluster/templates/monitor-dashboard-configmap.yaml]: how to review this long string?

**Comment by tennix** [comment]
Could you remove the pushgateway and use tikv metrics pull directly?

**Comment by xiaojingchen** [comment]
should we add these source json files and generation script to the repository?

**Comment by qiffang** [comment]
This ticket is duplicated with remove-pushgateway, so close it.

**Pull request closed without merging**